cmd/hub/main.go

```diff
@@ -69,8 +67,15 @@ func main() {
 		h.Close()
 	}()
 
+	go func() {
```

Comment:
I'd rather not to overload the main function by decomposing into a separate one. Moreover the same code is in several places.